examples/dataflow-xmlio-to-bq/README.md

```diff
@@ -0,0 +1,64 @@
+# XML to BQ via Cloud DataFlow
+This example contains a Dataflow pipeline to read nested XML data from Google Cloud Storage and write to BigQuery while maintaining the nested structure of the XML data in BQ via STRUCT fields. The example utilizes Beam's XMLIO to read XML input into a PCollection, creates BQ TableRow objects and writes them into BigQuery.
+
+## Requirements
+- Java 8.0.292
+- Maven 3.8.3
+
+## Building the pipeline
+Using maven, you can simply build the target jar via `mvn package`
+
+## Running the pipeline
+
+```
+export GOOGLE_APPLICATION_CREDENTIALS="/path/to/application-credentials.json" # If running locally
```

Comment:
Exporting a key file is not necessary if you use `gcloud auth application-default login` to use your personal account, or `gcloud config set auth/impersonate_service_account SERVICE_ACCOUNT` (assuming you have permission to use that service account). By using any of the above options, you don't have to export credentials, which are always a risk, and it is against the Google Cloud recommendations (https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf).

Please could you change that line and include a paragraph explaining that the user should have `gcloud` installed, and use any of the above commands? (and possibly that it would work "out of the box" if they are using the Cloud Shell).

Reply:
Thanks for the feedback. Ah yes good point, sorry I missed that. README updated.